xbmc/network/upnp/UPnPRenderer.cpp

```diff
@@ -251,8 +251,12 @@ void CUPnPRenderer::Announce(ANNOUNCEMENT::AnnouncementFlag flag,
 
     if (message == "OnPlay" || message == "OnResume")
     {
-      avt->SetStateVariable("AVTransportURI", g_application.CurrentFile().c_str());
-      avt->SetStateVariable("CurrentTrackURI", g_application.CurrentFile().c_str());
+      const std::shared_ptr<CFileItem> item = CServiceBroker::GetDataCacheCore().GetFileItem();
+      if (item)
+      {
+                avt->SetStateVariable("AVTransportURI", item->GetPath().c_str());
+                avt->SetStateVariable("CurrentTrackURI", item->GetPath().c_str());
```

Comment:
Indentation looks wrong.

Reply:
I'll fix that later in bulk, the file is completely fucked up

Reply:
alright